docs/pvcscale.md

```diff
@@ -0,0 +1,197 @@
+
+### PVCSCALE_MAXPODS
+Default: `1`
+Maximum number of pods that will be created by the pvcscale workload. 
+Typically much higher values are used than the default for large scale tests.
+PVCScale does not have implemented mechanism to calculate maximum number of pods which all allowed to be started on 
+particular cluster, so it is necessary to adapt `PVCSCALE_MAXPODS` to cluster capacity and
+execute it accordingly. 
+Choosing `PVCSCALE_MAXPODS` to be high can lead that many PODS/PVCs are created what can affect cluster limits and peformance.
+
+### PVCSCALE_POD_IMAGE
+Default: `quay.io/openshift-scale/scale-ci-workload`
+Pod image to use for pods that are pvscale pods. This image will be used to
+start POD and bound PVC to it.
+
+### PVCSCALE_STEPSIZE
+Default: `1`
+Number of PODS for cluster loader will create before waiting for pods to become running.
+PVCScale test creates one pod/pvc at time.
+
+### PVCSCALE_PAUSE
+Default: `0`
+Period of time (in seconds) for cluster loader to pause after creating pods and 
+waiting for them to be running.PVCScale does not wait on pods to be `Ready` 
+before it creates new ones - it creates pods in fastest possible manner. 
+
+### PVCSCALE_STORAGE_SIZE
+Default: `1Gi`
+This is default PVC size. If specified as `1Gi` then all pods will get `1Gi`
+device mounted inide pods.
+
+### PVCSCALE_STORAGECLASS
+Default: `none`
+It is important to have working storage class prior starting PVCScale test.
+`PVCSCALE_STORAGECLASS` parameter is name of storage class which one plan to
+use, and any storage class supporting dynamic storage provisioning will work
+
+### ACCESS_MODES
+Default: `ReadWriteOnce`
+`ACCESS_MODES` is access_mode for PVC inside container. This field does not
+affect the test itself and might be removed in future. 
+
+
+### Smoke test variables
+
+```
+PVCSCALE_TEST_PREFIX=pvcscale_smoke
+PVCSCALE_CLEANUP=true
+PVCSCALE_BASENAME=pvcscale
+PVCSCALE_MAXPODS=10
+PVCSCALE_POD_IMAGE="quay.io/openshift-scale/scale-ci-workload"
+PVCSCALE_STEPSIZE=1
+PVCSCALE_PAUSE=0
+```
+
+An longer example of variables can be found
+[here](https://gist.githubusercontent.com/ekuric/6038cb7437d58679de00d31b0365ce4c/raw/d7570f0a4499c6f9d9a395ecdd969bf3bc8b59b3/gistfile1.txt)
+
+### Test results 
+PVCScale test creates first the number of PODS specified in `PVCSCALE_MAXPODS`
+variable. Once all PODS are in `READY` state with PVC in `Bound` status,  PVCScale test 
+collects data for all PODS when they are started, and from this information we can then see
+how long it takes to start PODS from pod1..podN.
+
+The output will be like one below 
+```
+Pod-Ready
+07:56:29
+07:56:30
+07:56:31
+07:56:32
+07:56:35
+07:56:36
+07:56:39
+07:56:39
+07:56:44
+07:56:46
+``` 
+In this example are visible times when PODS were in `READY` state, where the
+first pod is started at `07:56:29` and last one at `07:56:46`.The time in this output
+is day time when test was executed, and this will be changed in future versions
```

Comment:
is a local time.
That's it.  I would not comment on changing anything in future versions here, even though you will.  If you insist, you have a Future work section.